modAionImpl/src/org/aion/zero/impl/cli/Cli.java

```diff
@@ -94,10 +101,24 @@ public int call(final String[] args, final Cfg cfg) {
                     }
                     break;
                 case "-c":
+                    if (args.length == 2 && isValid(args[1])) {
+                        switch (args[1].toLowerCase()) {
+                            case "mainnet":
+                            case "conquest":
+                                CfgAion.setConfFilePath(BASE_PATH + "/config/" + args[1] + "/config.xml");
+                                System.out.println("\nNew config generated for " + args[1]);
+                                break;
+                            default:
+                                System.out.println("\nInvalid network '" + args[1] + "'; Set to default mainnet");
+                                break;
+                        }
+                    } else if (args.length == 1) {
+                        System.out.println("\nInvalid arguments; Set to default mainnet");
```

Comment:
thats true...maybe for invalid networks and invalid datadir i should output some kind of guidance message instead of defaulting to the default network/path